src/App.tsx

```diff
@@ -1,61 +1,124 @@
+import React, { useEffect, useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
```

Comment:
Consider removing commented out code to improve readability and maintainability. 🧹